Samples/src/FSharpWithQSharp/QuantumCode/Operations.qs

```diff
@@ -0,0 +1,77 @@
+﻿namespace QuantumCode
+{
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+
+    // The oracle: f(x₀, …, xₙ₋₁) = Σᵢ (rᵢ xᵢ + (1 - rᵢ)(1 - xᵢ)) modulo 2 for a given bit vector r = (r₀, …, rₙ₋₁).
+    // Inputs:
```

Comment:
Would you please format this as a /// comment to enable hover support within VS Code and ? support within IQ#? Thanks!